document/document.go

```diff
@@ -487,6 +487,64 @@ func (d *Document) Paragraphs() []Paragraph {
 	return ret
 }
 
+// HasFootnotes returns a bool based on the presence or abscence of footnotes within
+// the document
+func (d *Document) HasFootnotes() bool {
+	if d.footNotes == nil {
+		return false
+	}
+	return true
+}
```

Comment:
it is ok but why just not to do it by one-liner:`return d.footNotes != nil` ? Also can it be the case that d.footNotes is an empty array, should this function return `true` or `false` in this case?

Reply:
👍